js/gameview.js

```diff
@@ -1,11 +1,22 @@
 // React view of a GameState
 import React from 'react';
 import Player from './playerview.js';
+import Card from "./cardview.js"
 
 
 let GameView = React.createClass({
     render() {
     window.client.gameView = this; 
+    let opponentBoardCards = this.props.state.players[1].board.map(function (cardInfo, i) {
+      return (
+          <Card cardInfo={cardInfo} player={this.props.state.players[1]} key={i} />
+      );
+    }.bind(this));
```

Comment:
you should look at how => functions ("fat arrow") work in ES6, you might not need these bind statements